OptionSelector/CQCategory.h

```diff
@@ -0,0 +1,17 @@
+//
+//  CQCategory.h
+//  Glycemic Index of Common Foods OptionSelector App
+//
+//  Created by Ayuna Vogel on 8/10/15.
+//  Copyright (c) 2015 Ayuna Vogel. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface CQCategory : NSObject
```

Comment:
For this one, I'd do

    /**
     * This represents a category to be shown on the main screen. It stores the
     * category's name, a list of the possible values under that category, and the
     * currently selected item in the category. This data is initialized and used
     * in MainTableViewController.
     */

That might be a bit more than is needed for something simple like this, but it should give you some idea of what I mean.

If you put the two stars at the top (like I did above), the doc text will even show up elsewhere as the description for the class, which can be really helpful when you've got a lot of classes:

![screen shot 2015-08-22 at 9 58 55 pm](https://cloud.githubusercontent.com/assets/782920/9426560/31666186-4919-11e5-8a29-9559bb9b5b9b.png)
![screen shot 2015-08-22 at 9 59 28 pm](https://cloud.githubusercontent.com/assets/782920/9426561/316a7e7e-4919-11e5-80b6-0f7f663ab841.png)